### DISCUSSION TITLE
Select which devices to show

### DISCUSSION THREAD

**Discussion by krazylenni** [hassio-addons/app-aircast#159] [Q&A] [answered]
I have a few devices in the home, but I don't need all of them showing. I tried to edit the xml file, but they simply come back everytime I reboot HA. Any suggestions?

**Comment by frenck** [comment, marked as answer]
Mark them as disabled in the XML config file.

`<enabled>0</enabled>`

See also: <https://github.com/philippe44/AirConnect#config-file-parameters>

../Frenck

**Reply by krazylenni** [reply]
Wow! Thank you! Really appreciated
